src/main/java/tc/oc/pgm/listeners/PGMListener.java

```diff
@@ -304,4 +306,24 @@ public void nerfFishing(PlayerFishEvent event) {
       }
     }
   }
+
+  @EventHandler(priority = EventPriority.MONITOR)
+  public void dropItemsOnQuit(PlayerParticipationStopEvent event) {
+    MatchPlayer quitter = event.getPlayer();
+    Player player = Bukkit.getPlayer(quitter.getId());
+
+      for (ItemStack item : quitter.getInventory().getContents()) { //Drop inventory
+        if (item != null) {
```

Comment:
You can structure this similar to how I did the armor below. You also need to check that the `item` is not `AIR`.

Reply:
Ok!